app.js

```diff
@@ -70,18 +103,51 @@ class Screen {
     this.header.classList.add('Header');
     this.header.innerHTML = `
     <div class="Header-item">
-      <a href="/" class="Header-link f4 d-flex flex-items-center">
+      <a class="Header-link f4 d-flex flex-items-center active" data-target="#list">
         <span>Awesome Books</span>
       </a>
+    </div>
+      <div class="Header-item Header-item--full"></div>
+      <div class="Header-item mr-0">
+        <a class="Header-link mr-3 active" data-target="#list">Book List</a>
+      </div>
+    <div class="Header-item mr-0">
+      <a class="Header-link mr-3" data-target="#newBook">Add new</a>
+    </div>
+    <div class="Header-item mr-0">
+      <a class="Header-link mr-3" data-target="#contact">Contact</a>
     </div>
     `;
 
-    this.layoutMain = document.createElement('div');
+    this.clock = document.createElement('div');
+    this.clock.classList.add('clock');
+    this.clock.innerHTML = `
+    ${Date()}
```

Comment:
- [ ] For setting the time you were supposed to use the luxon library. However, I have noticed that the link is not working on the project requirements but we apologize for the inconvenience. Please refer to this [article](https://moment.github.io/luxon/#/) on how to set it up as the broken link gets updated. Unfortunately, this is a hard requirement for the project and needs to be implemented.